app/controllers/application_controller/ci_processing.rb

```diff
@@ -978,6 +970,30 @@ def process_elements(elements, klass, task, display_name = nil, order_field = ni
 
   private ############################
 
+  def process_element_destroy(element, klass, name)
+    return unless element.respond_to?(:destroy)
+
+    audit = {:event        => "#{klass.name.downcase}_record_delete",
+             :message      => "[#{name}] Record deleted",
+             :target_id    => element.id,
+             :target_class => klass.base_class.name,
+             :userid       => session[:userid]}
+
+    model_name  = ui_lookup(:model => klass.name) # Lookup friendly model name in dictionary
+    record_name = get_record_display_name(element)
+
+    element.destroy
+
+    if element.destroyed?
+      AuditEvent.success(audit)
+      add_flash(_("%{model} \"%{name}\": Delete successful") % {:model => model_name, :name => record_name})
+    else
+      error_msg = element.errors.collect { |_attr, msg| msg }.join(';')
+      add_flash(_("%{model} \"%{name}\": Error during delete: ") %
+               {:model => model_name, :name => record_name} << error_msg, :error)
```

Comment:
Yes, even the presence of error message placeholder can have a value to the translating person,
so probably a little better way to do it would be:

```
add_flash(_("%{model} \"%{name}\": Error during delete: %{error_msg}") %
         {:model => model_name, :name => record_name, :error_msg => error_msg}, :error)
```